test/localization/spot_check_about_has_strings.js

```diff
@@ -0,0 +1,21 @@
+/*
+ * spot check that each language has values for the string id keys on About page
+ * that are contained in English (i.e. make sure strings will show up, not ids")
+ */
+var tap = require('tap');
+var languages = require('../../languages.json');
+window = {};
+require('../../intl/about.intl.js');
```

Comment:
I agree that it is hacky, and am +1 on eventually having build-locales be able to return whatever variable you want rather than `window._messages`. One thing that came up for me during this, though, is that, being that this is meant to be a spot check of what will be used in the system, does it make sense to use what will be built by the `make` process?